apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -12,55 +13,90 @@ import SelfPacedProgressTable from './SelfPacedProgressTable';
 import HeaderBannerNoImage from '@cdo/apps/templates/HeaderBannerNoImage';
 import TwoColumnActionBlock from '@cdo/apps/templates/studioHomepages/TwoColumnActionBlock';
 import ActionBlocksWrapper from '@cdo/apps/templates/studioHomepages/ActionBlocksWrapper';
+import CoteacherInviteNotification from '@cdo/apps/templates/studioHomepages/CoteacherInviteNotification';
+import OwnedSections from '@cdo/apps/templates/teacherDashboard/OwnedSections';
+import SetUpSections from '@cdo/apps/templates/studioHomepages/SetUpSections';
+import AddSectionDialog from '@cdo/apps/templates/teacherDashboard/AddSectionDialog';
 import style from './landingPage.module.scss';
 import './tableStyles.scss';
 import Tabs from '@cdo/apps/componentLibrary/tabs';
+import {
+  asyncLoadSectionData,
+  asyncLoadCoteacherInvite,
+  hiddenPlSectionIds,
+} from '@cdo/apps/templates/teacherDashboard/teacherSectionsRedux';
 
-const getAvailableTabs = () => {
-  // [TODO]: return a subset of the tabs below based on the user's permission level
-  return [
+const getAvailableTabs = permissions => {
+  let tabs = [
     {
       value: 'myPL',
-      text: i18n.plLandingHeading(),
+      text: i18n.professionalLearning(),
     },
-    // {
-    //   value: 'myFacilitatorCenter',
-    //   text: i18n.plLandingTabFacilitatorCenter(),
-    // },
-    // {
-    //   value: 'myRPCenter',
-    //   text: i18n.plLandingTabRPCenter(),
-    // },
-    // {
-    //   value: 'myWorkshopOrganizerCenter',
-    //   text: i18n.plLandingTabWorkshopOrganizerCenter(),
-    // },
-    // {
-    //   value: 'myInstructorCenter',
-    //   text: i18n.plLandingTabInstructorCenter(),
-    // },
   ];
+
+  if (permissions.includes('facilitator')) {
+    tabs.push({
+      value: 'myFacilitatorCenter',
+      text: i18n.plLandingTabFacilitatorCenter(),
+    });
+  }
+
+  if (
+    permissions.includes('universal_instructor') ||
+    permissions.includes('plc_reviewer')
+  ) {
+    tabs.push({
+      value: 'instructors',
+      text: i18n.plLandingTabInstructors(),
+    });
+  }
+
+  // {
+  //   value: 'myRPCenter',
+  //   text: i18n.plLandingTabRPCenter(),
+  // },
+  // {
+  //   value: 'myWorkshopOrganizerCenter',
+  //   text: i18n.plLandingTabWorkshopOrganizerCenter(),
+  // },
+  // {
```

Comment:
nit: should this one still be commented out?

Reply:
I think since we haven't set up any content yet for those tabs I've been leaving them commented out until they are useful. Otherwise Regional Partners and Workshop Organizers will have empty tabs